pages/changelog.mdx

```diff
@@ -135,7 +134,7 @@ Keep up to date on new product features and enhancements, API docs, tools change
 
 **ADDED**
 
-- Added a new funding source type for cards to support Push-to-Debit payments. Learn more about [Debit Cards](/concepts/debit-cards) and [Adding a Debit Card as a Funding Source](/guides/dwolla-cards-js/add-a-debit-card) in our documentation.
+- Added a new funding source type for cards to support Push-to-Debit payments.
```

Comment:
Can we put something here indicating that this product is no longer in service? Doubt this page is often read, but thinking of someone possibly reading this and trying to search/find documentation for it.

Reply:
Thought about that as well. Maybe just say `**Sunset Notice:** this feature was sunset on x date"`

Reply:
Agreed. Changes are in this [commit](https://github.com/Dwolla/dev-portal/pull/517/commits/bbbc694080233656cf5e7f475d56b3850130b99e).